src/diem/testing/cli/click.py

```diff
@@ -89,6 +120,8 @@ def test(
         os.environ[envs.DMW_STUB_BIND_PORT] = str(stub_bind_port)
     if stub_diem_account_base_url:
         os.environ[envs.DMW_STUB_DIEM_ACCOUNT_BASE_URL] = stub_diem_account_base_url
+    if import_stub_diem_account_config_file:
```

Comment:
man we need a better way to pass these along

Reply:
any better idea?